index.js

```diff
@@ -87,6 +94,9 @@ function fastifyOauth2 (fastify, options, next) {
   const tags = options.tags || []
   const schema = options.schema || { tags }
   const cookieOpts = Object.assign({ httpOnly: true, sameSite: 'lax' }, options.cookie)
+  const userAgent = options.userAgent
```

Comment:
it should be possible to disable it

Reply:
@mcollina I added the possibility for an explicit `false` that disables it